src/hits/__tests__/hits.spec.ts

```diff
@@ -87,4 +87,40 @@ describe('Hits', () => {
     const fixture = render({});
     expect(fixture).toMatchSnapshot();
   });
+  it('should allow calling insightsClient', () => {
+    const render = createRenderer({
+      defaultState,
+      template: `
+          <ais-hits>
+            <ng-template let-hits="hits" let-insights="insights">
+              <ul>
+                <li *ngFor="let hit of hits">
+                  <button 
+                    id="add-to-cart-{{hit.objectID}}" 
+                    (click)="insights('clickedObjectIDsAfterSearch', { eventName: 'Add to cart', objectIDs: [hit.objectID] })">
+                    
+                  </button>
+                </li>
+              </ul>
+            </ng-template>
+          </ais-hits>
+        `,
+      TestedWidget: NgAisHits,
+      additionalDeclarations: [NgAisHighlight],
+    });
+
+    const insights = jest.fn();
+    const fixture = render({ insights });
+    expect(fixture).toMatchSnapshot();
```

Comment:
The snapshot is not really useful for this assertion, is it?

Reply:
👍  I've removed it